plugins/aws/resoto_plugin_aws/accountcollector.py

```diff
@@ -1787,13 +1811,15 @@ def collect_albs(self, region: AWSRegion, graph: Graph) -> None:
                 v = graph.search_first("id", vpc_id)
                 if v:
                     graph.add_edge(v, a)
+                    graph.add_edge(v, a, edge_type=EdgeType.delete)
```

Comment:
The direction of the edge is correct here, right?

Reply:
Yes, sometimes the logical direction is equal to the delete direction (e.g. VPC -> Subnet, first delete the Subnet, then the VPC). Sometimes it's reversed (e.g. EBS volume -> instance, first unmount or delete the instance, then the storage) and sometimes there is only a logical and no delete edge.

Reply:
👍